CHANGELOG.md

```diff
@@ -32,9 +44,6 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 - Added documentation to the remaining exported types. (#24)
 
-- Changed version of `github.com/iver-wharf/wharf-api-client-go`
-  from v1.2.0 -> v1.3.1. (#26)
-
```

Comment:
This removal should be reverted. My guess is that it was from before v1.3.0 was released.